packages/wordpress-playground-block/src/components/playground-preview/index.tsx

```diff
@@ -634,11 +685,11 @@ export default function PlaygroundPreview({
 						}
 					</span>
 				</div>
-			</section>
-			<footer className="demo-footer">
+			</div>
+			<footer className="wordpress-playground-footer">
```

Comment:
Move the footer to be a child of the `<section>`. It was incorrectly outside of the section and needs to be part of the section for styling in this PR.